doc/user_guide/user_guide.md

```diff
@@ -2,84 +2,23 @@
+## Current issues with the connector
 
-(installation instructions found here : https://docs.microsoft.com/en-us/connectors/custom-connectors/paconn-cli ) 
+"Connect via on-premises data gateway" is currently NOT supported.  
+A separate connector and instructions will be delivered on a later date if there is demand for this.
 
-The connector talks to the exasol-rest-api which you'll also need to setup. See the section above.
-
-### Getting the connector files
-
-Clone this repository, download the repository zip or download the latest release from github and unzip the files.
-The connector consists of a settings file, an icon file and a apiDefinition.swagger.json and apiProperties.json file which describe the connector.
-
-### Using the paconn CLI tool to upload/create the connector within your organisation.
-
-Open up the command line or a terminal.
-
-###### Authenticate
-
-First you'll need to authenticate in paconn, type:
-`paconn login`
-Follow the steps to succesfully authenticate.
-
-###### Creating the connector
-
-Next you need to navigate to the folder where you downloaded or unzipped the connector files.
-
-Run the following command:
-`paconn create -s [Path to settings.json]`
-You'll be prompted to select a Power Apps environment. 
-
-Pick the environment where you want to install the connector.
-Follow the steps.
-If you get prompted to alter settings.json, press "yes". 
-
-( It will update the id of the connector so you can easily update it afterwards using `paconn update -s [Path to settings.json]`).
-
-###### Inspecting the connector we just created in our environment/organisation.
-
-If you now browse to https://make.powerapps.com/ , pick the right organisation top right and click on the "Dataverse" tab to the left and then "Custom Connectors" you should be able to see the connector.
-
-![image-20211122114520362](user_guide.assets/image-20211122114520362.png)
-
-###### Configuring the connector
-
-You can further configure the connector via the edit icon if you so wish (this is an optional step):
-
-![image-20211122115800918](user_guide.assets/image-20211122115800918.png)
-
-Here you can: 
-
-- Enable HTTPS in case of the HTTP version.
-  Before you do so: There's also a HTTPS version of the connector available for download.
-
-![image-20211122114543409](user_guide.assets/image-20211122114543409.png)
-
-Enabling HTTPS is a 2 step process in our connector's case, you'll also need to alter the host URL policy's Url template.
-
-Change "http" into "https". 
-
-
-
-![image-20211125134028331](user_guide.assets/image-20211125134028331.png)
-
-
-
-
-
-Important: Enabling "Connect via on-premises data gateway" is currently NOT supported. A separate connector and instructions will be delivered on a later date.
-
-### Making a connection using the connector
+## Making a connection using the connector
```

Comment:
Should we remove this section since it is already in local installation doc?

Reply:
These will probably differ somewhat also it makes for more coherent reading, since it's also required for the testing step (which you won't have with the gallery version)